dev/src/main/java/org/eclipse/codewind/intellij/ui/CodewindToolWindow.java

```diff
@@ -35,24 +35,7 @@
 import org.eclipse.codewind.intellij.core.connection.LocalConnection;
 import org.eclipse.codewind.intellij.core.connection.RemoteConnection;
 import org.eclipse.codewind.intellij.core.console.SocketConsole;
-import org.eclipse.codewind.intellij.ui.actions.AddExistingProjectAction;
-import org.eclipse.codewind.intellij.ui.actions.CloseAllLogsAction;
-import org.eclipse.codewind.intellij.ui.actions.DisableAutoBuildAction;
-import org.eclipse.codewind.intellij.ui.actions.EnableAutoBuildAction;
-import org.eclipse.codewind.intellij.ui.actions.InstallCodewindAction;
-import org.eclipse.codewind.intellij.ui.actions.OpenAppOverviewAction;
-import org.eclipse.codewind.intellij.ui.actions.OpenApplicationAction;
-import org.eclipse.codewind.intellij.ui.actions.OpenIdeaProjectAction;
-import org.eclipse.codewind.intellij.ui.actions.OpenPerformanceDashboardAction;
-import org.eclipse.codewind.intellij.ui.actions.OpenTektonDashboardAction;
-import org.eclipse.codewind.intellij.ui.actions.RefreshAction;
-import org.eclipse.codewind.intellij.ui.actions.RemoveProjectAction;
-import org.eclipse.codewind.intellij.ui.actions.ShowAllLogsAction;
-import org.eclipse.codewind.intellij.ui.actions.StartBuildAction;
-import org.eclipse.codewind.intellij.ui.actions.StartCodewindAction;
-import org.eclipse.codewind.intellij.ui.actions.StopCodewindAction;
-import org.eclipse.codewind.intellij.ui.actions.UninstallCodewindAction;
-import org.eclipse.codewind.intellij.ui.actions.UpdateCodewindAction;
+import org.eclipse.codewind.intellij.ui.actions.*;
```

Comment:
Probably better in this case to use the wildcard instead of listing each action.

Reply:
That change is the result of running IntelliJ's `Code > Optimize Imports` action.